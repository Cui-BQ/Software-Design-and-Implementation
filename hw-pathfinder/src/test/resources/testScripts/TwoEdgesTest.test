CreateGraph graph1

AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

AddEdge graph1 n1 n2 1.2
AddEdge graph1 n1 n2 0.12

AddEdge graph1 n2 n3 2.3
AddEdge graph1 n2 n3 0.23

FindPath graph1 n1 n3