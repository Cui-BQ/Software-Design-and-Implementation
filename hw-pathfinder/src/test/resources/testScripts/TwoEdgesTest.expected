created graph graph1

added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

added edge 1.200 from n1 to n2 in graph1
added edge 0.120 from n1 to n2 in graph1

added edge 2.300 from n2 to n3 in graph1
added edge 0.230 from n2 to n3 in graph1

path from n1 to n3:
n1 to n2 with weight 0.120
n2 to n3 with weight 0.230
total cost: 0.350